daemon/mgr/container_envs.go

```diff
@@ -25,33 +24,24 @@ import (
 // /etc/profild.d/*.sh (here is pouchenv.sh). After that a process execed from outside would take env of init process,
 // then such action like reload of detailed user application could share the env.
 const (
-	// containerInstanceInfo is file in the container image.
-	// row in this file has content like `env_appDeployType = taoJavaWeb`
-	containerInstanceInfo = "/etc/instanceInfo"
 	// pouchEnvDir =
 	pouchEnvDir = "/etc/profile.d"
 	// this filepath is used in PouchContainer to store user input env via persistent file
 	pouchEnvFile = "/etc/profile.d/pouchenv.sh"
 )
 
 // updateContainerEnv update the container's envs in
-// /etc/instanceInfo and /etc/profile.d/pouchenv.sh
-// Env used by rich container.
+// /etc/profile.d/pouchenv.sh used by rich container.
 func updateContainerEnv(inputRawEnv []string, baseFs string) error {
 	var (
-		envPropertiesPath = path.Join(baseFs, containerInstanceInfo)
-		envShPath         = path.Join(baseFs, pouchEnvFile)
```

Comment:
Where do we create `envShPath` and `envShDir`, are they create in rich-mode with runc?

Reply:
The file is located in the image's rootfs. 
In addition, the file is in the base image.

Reply:
do you mean we not create the file, but only judge the file

Reply:
yes